Slim/Plugin/ExtendedBrowseModes/strings.txt

```diff
@@ -324,12 +324,12 @@ PLUGIN_EXTENDED_BROWSEMODES_TAGS
-	FR	Indiquez à LMS tous les tags de rôles (toutes les contributions) à utiliser dans la bibliothèque musicale. Par exemple, SOLOIST (soliste), DRUMMER (batteur), PRODUCER (producteur), etc. Ces rôles peuvent ensuite être utilisés dans les modes de navigation supplémentaires ci-dessus. Ils apparaîtront également en tant que catégories lorsque vous parcourrez les albums ou les artistes en activant le regroupement par type de sortie. Les modifications apportées ici déclencheront une nouvelle analyse de la bibliothèque musicale.
-	ES	Informe a LMS sobre cualquier etiqueta de rol de colaborador personalizada utilizada en la biblioteca musical. Por ejemplo, SOLISTA, BATERISTA, PRODUCTOR, etc. Estas etiquetas se pueden utilizar como roles de colaborador en los modos de exploración adicionales anteriores. También aparecerán como categorías al explorar álbumes o artistas con la agrupación Tipo de publicación activada. Los cambios que se realicen aquí provocarán una nueva exploración de la biblioteca musical.
-	HU	Mondja el az LMS-nek a zenei könyvtárban használt egyéni közreműködői szerepcímkéket. Például SZÓLISTA, DOBOS, PRODUCER stb. Ezek ezután közreműködői szerepként használhatók a fenti További böngészési módokban. Kategóriaként is megjelennek, ha olyan albumokat vagy előadókat böngész, amelyeknél a Kiadástípusok csoportosítása engedélyezett. Az itt végzett változtatások elindítják a zenei könyvtár újraellenőrzését.
-	NL	Vertel LMS over alle uitvoerende rollen die je hebt getagged in jouw muziekbibliotheek. Bijvoorbeeld SOLOIST, DRUMMER, PRODUCER, enz. Deze kunnen dan worden gebruikt als uitvoerende rollen in de additionele blader modi hierboven. Ze verschijnen ook als categorieën wanneer je door albums of artiesten bladert met de Release Type groepering ingeschakeld. Als je hier wijzigingen aanbrengt, wordt de muziekbibliotheek opnieuw gescand.
+	EN	Tell LMS about any custom contributor role tags used in the music library. For example, SOLOIST, DRUMMER, PRODUCER, etc. These can then be used as Contributor Roles in the Additional Browse Modes above. Optionally, include the roles in the combined Artists list or the All Artists list and in Album list links. Adding, deleting or changing Role Tags will trigger a rescan of your music library. The display text and other options can be changed without a rescan.
+	DE	Informieren Sie LMS über alle benutzerdefinierten Rollen-Tags, die in der Musikbibliothek verwendet werden. Zum Beispiel SOLOIST, DRUMMER, PRODUCER, usw. Diese können dann als Mitwirkendenrollen in den oben genannten zusätzlichen Suchmodi verwendet werden. Optional können Sie die Rollen in die kombinierte Künstlerliste oder die Liste „Alle Künstler“ und in die Links zur Albumliste aufnehmen. Das Hinzufügen, Löschen oder Ändern von Rollen-Tags löst einen erneuten Scan Ihrer Musikbibliothek aus. Der Anzeigetext und andere Optionen können ohne erneuten Scan geändert werden.
```

Comment:
"Definiere benutzerdefinierte Rollen-Tags, die in der Musikbibliothek verwendet werden. Zum Beispiel SOLOIST, DRUMMER, PRODUCER, usw. Diese können dann als weitere Künstler in den oben genannten zusätzlichen Suchmodi verwendet werden. Optional können Sie die Rollen in die kombinierte Künstlerliste oder die Liste "Alle Künstler" und in die Links zur Albumliste aufnehmen. Das Hinzufügen, Löschen oder Ändern von Rollen-Tags erfordert einen erneuten Scan Ihrer Musikbibliothek. Der angezeigte Text und andere Optionen können ohne erneuten Scan geändert werden."